src/trivia/config.h.cmake

```diff
@@ -61,6 +61,13 @@
  * showing fiber call stack.
  */
 #cmakedefine ENABLE_BACKTRACE 1
+
+/*
+ * Defined if configured with ENABLE_BACKTRACE on AARCH64 and
+ * '-mbranch-protection' compile flag is passed.
+ */
+#cmakedefine ENABLED_BRANCH_PROTECTION
```

Comment:
I don't like that we have `ENABLE_` and `ENABLED_` prefixes - this is confusing. Maybe, name it `HAVE_BRANCH_PROTECTION`?

Reply:
IMO, the `HAVE_` prefix is used for marking availability of compiler options.

Whereas here we have to deal with the case that a specific compiler option _was passed_, i.e., a specific compiler feature is enabled.

Reply:
Who sets this option? I don't see any strings matching `branch-protection` in the source.

Reply:
Discussed offline:
- The flag can be set by some distributions (e.g. Fedora).
- `ENABLED_` name is fine.